rails_application/app/controllers/customers_controller.rb

```diff
@@ -24,6 +24,13 @@ def update
     redirect_to customers_path, notice: "Customer was promoted to VIP"
   end
 
+  def show
+    @customer = Customers::Customer.find(params[:id])
+    @customer_orders = ClientOrders::Order.where(client_uid: params[:id])
```

Comment:
I'm not sure if I can use this read model because I think it was created for something else. There is also Orders::Order read model which I think is better suited for this use case but it doesn't have customer_id. 

Please let me know if the current implementation is fine or I should add customer_id to Orders::Order or maybe implement another read model.